### PR TITLE
Separate out event page

### PR DESCRIPTION
**Pull request by ShrawanBK**

### CODE REVIEW

**Review by tnagorra — CHANGES_REQUESTED** [review]
We should re-use the Events Table in Crisis and Events page

**Review by frozenhelium — APPROVED** [review]

File: src/views/Events/index.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    const {
+        className,
+    } = props;
+
+    return (
+        <div className={_cs(styles.events, className)}>
+            <PageHeader
+                title="Events"
+            />
+            <EventsTable />
```
Comment (tnagorra) [inline src/views/Events/index.tsx:22]: The styling for EventsTable
```
    .container {
        margin: var(--spacing-medium);
        min-height: 400px;
    }

```

File: src/views/Crisis/index.tsx [hunk 2]
```diff
@@ -354,68 +57,15 @@ function Crisis(props: CrisisProps) {
...
-                            defaultCrisis={crisisData?.crisis}
-                        />
-                    </Modal>
-                )}
-            </Container>
+            <>
+                <PageHeader
+                    title="Events"
+                />
+                <EventsTable
```
Comment (tnagorra) [inline src/views/Crisis/index.tsx:64]: Pass `className={styles.largeContainer}`

File: src/components/EventsTable/styles.css [hunk 3]
```diff
@@ -0,0 +1,29 @@
+.container {
+    margin: var(--spacing-medium);
+    min-height: 200px;
+}
+
+.large-container {
+    margin: var(--spacing-medium);
+    min-height: 400px;
+}
+
```
Comment (tnagorra) [inline src/components/EventsTable/styles.css:10]: This should not be on EventsTable

File: src/components/EventsTable/index.tsx [hunk 4]
```diff
@@ -0,0 +1,411 @@
...
+    name: 'createdAt',
+    direction: TableSortDirection.dsc,
+};
+
+const keySelector = (item: EventFields) => item.id;
+
+interface EventsProps {
+    className?: string;
+    defaultCrisis?: CrisisOption | null;
+    showCrisisColumn?: boolean;
```
Comment (tnagorra) [inline src/components/EventsTable/index.tsx:114]:
```suggestion
    cirsisColumnShown?: boolean;
```

**Pull request merged**